### PR TITLE
[helm][tf] update default resources and machine types

### PR DESCRIPTION
**Pull request by sherry-x**
### Description
- Update resources based on https://aptos.dev/nodes/ait/node-requirements
- Update instance types
- did not update testnet deployment yet, since it might still make sense to run smaller instance there

### Test Plan

TODO deploy on multiple clouds

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3005)

### CODE REVIEW

File: terraform/helm/aptos-node/values.yaml [hunk 1]
```diff
@@ -26,16 +26,16 @@ validator:
...
-      cpu: 3.5
-      memory: 6Gi
+      cpu: 15.5
+      memory: 30Gi
   storage:
     # -- Kubernetes storage class to use for validator persistent storage
     class:
     # -- Size of validator persistent storage
-    size: 350Gi
+    size: 512Gi
```
Comment (grao1991) [inline terraform/helm/aptos-node/values.yaml:38]: I thought we agreed on 1T?

Comment (sherry-x) [reply terraform/helm/aptos-node/values.yaml:38]: @grao1991 will update this after AIT3. but for the 10 days 512 should be good enough

File: terraform/aptos-node/aws/variables.tf [hunk 2]
```diff
@@ -142,7 +142,7 @@ variable "helm_enable_validator" {
 
 variable "utility_instance_type" {
   description = "Instance type used for utilities"
-  default     = "t3.medium"
+  default     = "t4g.medium"
```
Comment (grao1991) [inline terraform/aptos-node/aws/variables.tf:145]: Are we going to run prometheus on this? I saw it sometimes consume a lot of RAM and get OOM killed.

Comment (sherry-x) [reply terraform/aptos-node/aws/variables.tf:145]: single node deployment doesn't have this problem, you can override the instance type for the testnet terraform

**Review by grao1991 — APPROVED** [review]

**Pull request merged**